ccd-definition/AuthorisationCaseField/CareSupervision/barrister.json

```diff
@@ -357,5 +357,12 @@
     "CaseFieldID": "correspondenceDocumentsLANC",
     "UserRole": "[BARRISTER]",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "additionalApplicationsBundle",
+    "UserRole": "[BARRISTER]",
+    "CRUD": "RU"
```

Comment:
should not it be just R?

Reply:
I applied the same access permissions as existing `c2DocumentBundle`

Reply:
`c2DocumentBundle.supportingEvidenceDocument` field has access permissions as `U` in See the existing c2DocumentBundle authorisation complex fields [here](https://github.com/hmcts/fpl-ccd-configuration/blob/master/ccd-definition/AuthorisationComplexType/caseDocuments/C2DocumentBundle.json#L313)